scope/src/doctor/runner.rs

```diff
@@ -53,9 +53,7 @@ impl Display for PathRunResult {
 impl PathRunResult {
     fn process(&mut self, group: &GroupExecutionResult) {
         let group_name = group.group_name.to_string();
-        if group.skip_remaining {
-            self.skipped_group.insert(group_name.clone());
-        }
+        //skip_remaining is handled up a level in the loop
```

Comment:
This is where the skips are handled.  
https://github.com/oscope-dev/scope/blob/419291a7b5074a5c88b430b989de06dadf647f30/scope/src/doctor/runner.rs#L146-L149